llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -3357,6 +3357,37 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     setOriginForNaryOp(I);
   }
 
+  Value *convertBlendvToSelectMask(IRBuilder<> &IRB, Value *C) {
```

Comment:
should these two be `static`?

Reply:
CreateAppToShadowCast can't be static

Reply:
they are member of MemorySanitizerVisitor:: and access some fields